react-ui/src/ProjectSlug.js

```diff
@@ -0,0 +1,17 @@
+import Button from 'react-bootstrap/Button'
```

Comment:
nit: _import { Button, Form } from "react-bootstrap";_

react-bootstrap imports could be combined into one import.